### ISSUE TITLE
Leaflet timeline

### ISSUE THREAD

**Issue by xbgmsharp** [DavidMStraub/gramps-webapp#33]
I created a new Leaflet map with timeline. 

Using GEOJSON as input, markers appears on birth date at birth place and disappears at death date.

To generate the GEOJSON, I created a python parser from /api/tree JSON object.

Using the demo tree input you can get the following output.

![Leaflet_timeline_demo](https://user-images.githubusercontent.com/1498985/81838033-e2e28700-9545-11ea-9368-b5e391025a97.gif)

**Comment by DavidMStraub** [comment]
Beautiful! Would you like to share your code? Even if it's not a pull request but just as a Gist or something like that would be interesting.

**Comment by xbgmsharp** [comment]
Definitely, to be honest I still not sure how to integrate the code with lit-html. 

Dependencies:
* Leaflet
* Leaflet Plugin, https://github.com/skeate/Leaflet.timeline

Python script to convert `/api/tree` JSON to GEOJSON. 
JS script to convert `/api/tree` JSON to GEOJSON. 

Not sure if it should be in the backend or the frontend?

https://gist.github.com/xbgmsharp/cdbfd2edb76d0528d02f3f9e657fe2c3
